lib/searchable_select.ex

```diff
@@ -48,12 +50,12 @@ defmodule SearchableSelect do
       socket
       |> assign(:search, "")
       |> assign(:disabled, assigns[:disabled])
-      |> assign(:placeholder, assigns[:placeholder])
+      |> assign(:placeholder, assigns[:placeholder] || "Search")
```

Comment:
Does this need to be defaulted? Won't it already be defaulted in the below `update`?

Reply:
because we're taking from assigns it seems that as soon as an event happens it'll hit the assigns with a nil placeholder and blank out the Search. 
![image](https://user-images.githubusercontent.com/122778195/219047489-525ea966-d9fc-4ff8-b91f-d5674072ffca.png)
You can see it on any searchable select currently on live